pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -48,6 +48,8 @@
 , withDav1d ? withHeadlessDeps # AV1 decoder (focused on speed and correctness)
 , withDc1394 ? withFullDeps && !stdenv.isDarwin # IIDC-1394 grabbing (ieee 1394)
 , withDrm ? withHeadlessDeps && (with stdenv; isLinux || isFreeBSD) # libdrm support
+, withDvdnav ? withFullDeps && withGPL && lib.versionAtLeast version "7" # needed for DVD demuxing
+, withDvdread ? withFullDeps && withGPL && lib.versionAtLeast version "7" # needed for DVD demuxing
```

Comment:
These aren't used anywhere. Shouldn't the new `enableFeature` calls check for them?

Reply:
That looks like a simple copy paste error ;)

Reply:
oops, fixed that should now not cause rebuild